wandb/wandb_run.py

```diff
@@ -661,6 +661,41 @@ def _add_viz(self, key, viz_id):
         }
         self.config.persist()
 
+    # Stores a singleton item to wandb config.
+    #
+    # A singleton in this context is a piece of data that is continually
+    # logged with the same value in each history step, but representing
+    # as a single item in the config.
+    #
+    # We do this to avoid filling up history with a lot of repeated uneccessary data
```

Comment:
It doesn't affect the history.

Reply:
That's exactly the point we're storing it here to keep it out of history. Is the wording here confusing?

Reply:
Oh I get it.